### PR TITLE
A tile right_angle_180 is added

### PR DESCRIPTION
**Pull request by Sky0023**
straight_0.png tile is added

### CODE REVIEW

**Comment by bhaumikmistry** [comment]
@Sky0023 Your spot for the tile was already taken by someone before you. we are following FIPRFM (First In Pull Request First Merge) model to avoid conflicts, Please rename your PR with the coordinate and use another tile to contribute.

**Comment by Sky0023** [comment]
okay

**Comment by Sky0023** [comment]
Please see this file I have updated it. The spot of the tile is changed

**Comment by Sky0023** [comment]
Please check the "A right_angle_90 tile is added" file . I have also changed the spot of the tile

File: docs/index.html [hunk 1]
```diff
@@ -86,6 +86,9 @@ <h1>Image Grid</h1>
 
 <div class="row-parent"> 
 <div class="row"> 
+<div class="row-parent">
```
Comment (bhaumikmistry) [inline docs/index.html]: @Sky0023 this is creating a conflict. Take a look.

**Comment by Sky0023** [comment]
I have resolved the conflict in master branch and created another branch and send you a pull request.

**Pull request merged**